src/find-best-model/src/main/scala/FindBestModel.scala

```diff
@@ -52,6 +78,16 @@ class FindBestModel(override val uid: String) extends Estimator[BestModel]
     if (trainedModels.isEmpty) {
       throw new Exception("No trained models to evaluate.")
     }
+    if (!(Set(MetricConstants.MseSparkMetric,
```

Comment:
Nit: might want to add this set to MetricConstants so that users who add a metric can have it show up

Reply:
done!